guides/source/routing.md

```diff
@@ -354,11 +383,10 @@ This will generate the exact same routes as the first example. You can also spec
 resources :articles, shallow: true do
   resources :comments
   resources :quotes
-  resources :drafts
 end
 ```
 
-The articles resource here will have the following routes generated for it:
+The articles resource above will generate the following routes:
```

Comment:
Open the 2nd one, don't feel strongly. Though it is accurate as is since `articles` (plural) is by rails convention and while the method in the routes files is `resources`, this sentence is referring to the word "resource" and not the method per se. But open to updating to the 2nd one is others feel strongly, fine either way.